server/data/data-library/src/main/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTable.java

```diff
@@ -308,6 +308,27 @@ public void removeForwardMapping(String user, Domain domain, String address) thr
         removeMapping(user, domain, mapping);
     }
 
+    @Override
+    public void addGroupMapping(String user, Domain domain, String address) throws RecipientRewriteTableException {
+        Mapping mapping = MappingImpl.group(address)
```

Comment:
copy/paste for this seems wrong as it's likely that someone change this code without changing it in remove function, right ?